gluten-ut/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenCastSuite.scala

```diff
@@ -20,4 +20,6 @@ package org.apache.spark.sql.catalyst.expressions
 import org.apache.spark.sql.{GlutenTestConstants, GlutenTestsTrait}
 
 class GlutenCastSuite extends CastSuite with GlutenTestsTrait {
+  override def blackTestNameList: Seq[String] = Seq.empty
```

Comment:
If the blacklist is empty, you can remove the definition, if not, we'd better to include super's blacklist in case that it'll override CH Backends' blacklist. For example, 
`
  override def blackTestNameList: Seq[String] = super.blackTestNameList ++ Seq("UT name")
`

Reply:
Thanks. This PR is not ready. It seems like my local dev env is broken and just want to test if it's able to run via GHA.